docs/stencil-docs/customizing-checkout/checkout-sdk-quickstart.mdx

```diff
@@ -1,61 +1,176 @@
-# Checkout SDK Overview
+# Getting Started with Checkout SDK

```

Comment:
Not sure if you want to continue with making the titles less gerund so take or leave. :)

Reply:
I decided to keep getting started because it seems to be used a lot in our docs.